onnx/optimizer/passes/fuse_conv_add_into_bias.h

```diff
@@ -0,0 +1,43 @@
+// ATTENTION: The code in this file is highly EXPERIMENTAL.
+// Adventurous users should note that the APIs will probably change.
+
+#pragma once
+
+#include "onnx/optimizer/passes/optimize_pass.h"
+
+namespace ONNX_NAMESPACE { namespace optimization {
+
+struct FuseConvAddIntoBias final : public OptimizePass {
+  explicit FuseConvAddIntoBias()
+    : OptimizePass("fuse_conv_add_into_bias", API_TYPE::IR) {
+  }
+
+  static int idx_of_conv(const onnx::ArrayRef<onnx::Value *> & values) {
```

Comment:
We use ONNX_NAMESPACE

Reply:
I use `namespace ONNX_NAMESPACE { namespace optimization {` at first. What should I do with this method?

Reply:
I think `onnx::` is not needed, ArrayRef should be visible. I will double check.

Reply:
checked and removed.